docs/service-mesh/05-05-production-profile.md

```diff
@@ -31,98 +31,9 @@ As the production profile is configured with increased performance it mind, the
 
 ## Use the production profile
 
->**CAUTION:** Due to changes in the installation options in Istio, Helm-based configuration is now deprecated in favor of the new IstioControlPlane API. Please keep in mind that Helm overrides will be no longer supported in future Istio releases. Refer to [IstioControlPlane](https://istio.io/docs/reference/config/istio.operator.v1alpha1/) documentation for details.
-
 You can deploy a Kyma cluster with Istio configured to use the production profile, or configure Istio in a running cluster to use the production profile. Follow these steps:
```

Comment:
Maybe it would be good to somehow link here the page `Istio custom configuration` you made as you are removing the section regarding Istio Control Plane API below.